libs/ActivityKit/include/ActivityKit.h

```diff
@@ -0,0 +1,27 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <unordered_map>
+
+#include "Activity.h"
+#include "MagicCard.h"
+namespace leka {
+
+class ActivityKit
+{
+  public:
+	explicit ActivityKit() = default;
+
+	void registerActivities(std::unordered_map<MagicCard, interface::Activity> const &activities);
+	void start(const MagicCard &card);
+	void stop();
+
+  private:
+	interface::Activity *_activity = nullptr;
+	std::unordered_map<MagicCard, interface::Activity> _activities {};
```

Comment:
ici plutôt un pointeur :) 

```suggestion
	std::unordered_map<MagicCard, interface::Activity*> _activities {};
```

Reply:
ça évite de copier les activities dans le map et ça gagne de l'espace.